### PR TITLE
Refactor header-cards component to use embedded snippet

### PR DESCRIPTION
**Pull request by neoformit**
Some refactoring of our "header-cards" and footer embedded snippets.

### CODE REVIEW

**Comment by neoformit** [comment]
@bebatut I'm not sure who should review this, but would you be able to make me a maintainer to this repo so I can review and merge Galaxy Lab PRs?

**Review by bebatut — APPROVED** [review]

**Pull request merged**
